charts/spire/charts/spire-server/templates/federation-ingress.yaml

```diff
@@ -1,6 +1,9 @@
 {{- if .Values.federation.enabled }}
 {{- if .Values.federation.ingress.enabled -}}
 {{- $svcName := include "spire-server.fullname" . }}
+{{- $path := "/"}}
+{{- $pathType := "Prefix" }}
+{{- $tlsSection := true }}
```

Comment:
Yeah, good catch.

I think there is one more issue to resolve and I think it looks good to me. I can't do an inline suggestion unfortunately.

Down a few lines (14 / 17) is a duplicate:
```
{{- $tlsSection := true }}
```
That should probably be removed.